lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -257,6 +257,15 @@ def test_lstchain_dl1_to_dl2(simulated_dl2_file):
     assert "reco_src_x" in dl2_df.columns
     assert "reco_src_y" in dl2_df.columns
 
+@pytest.mark.private_data
+def test_lstchain_find_pedestals(temp_dir_observed_files):
```

Comment:
I think you also should add the fixture to the actual files `observed_dl1_files` to assure the files are also produced
```
def test_lstchain_find_pedestals(temp_dir_observed_files, observed_dl1_files):
```